content/docs/setup/kubernetes/quick-start-ibm/index.md

```diff
@@ -69,69 +46,99 @@ Make sure to use the `kubectl` CLI version that matches the Kubernetes version o
     $ helm init --service-account tiller
     {{< /text >}}
 
-1. Add the IBM Cloud Helm repository to your Helm instance:
+### Deploy the Istio Helm charts
+
+1. Install the `istio-init` chart to bootstrap all the Istio CRDs:
 
     {{< text bash >}}
-    $ helm repo add ibm-charts https://registry.bluemix.net/helm/ibm-charts
+    $ helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system
     {{< /text >}}
 
-### Deploy the Istio Helm chart
-
-1. If using a Helm version prior to 2.10.0, install Istio’s Custom Resource Definitions via `kubectl apply`, and wait a few seconds for the CRDs to be committed
-to the Kubernetes API server:
+    To verify that all Istio CRDs were committed in the Kubernetes api-server, check the number of CRDs with the specified suffix with the following command. Verify that the number of total CRDs created was `58` for Istio:
 
     {{< text bash >}}
-    $ kubectl apply -f https://raw.githubusercontent.com/IBM/charts/master/stable/ibm-istio/templates/crds.yaml
+    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
+    58
     {{< /text >}}
 
 1. Install the Helm chart to your cluster:
 
+    {{< tip >}}
+    The Istio `demo` profile (`install/kubernetes/helm/istio/values-istio-demo.yaml`) is specified in the above command to support the IBM Cloud Kubernetes Service free cluster, which only contains a single worker providing fewer resources than needed.
+    If using a paid cluster of sufficient size, you can remove the `--values` parameter which will use the default Istio configuration values instead.
+    {{< /tip >}}
```

Comment:
This tip should be after the bash section. Or it needs to be changed to say "in the following command".

Reply:
Done.